eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/trace/Trace.java

```diff
@@ -140,7 +141,7 @@ public Span addTraceInfoToSpan(Span span, CloudEvent cloudEvent) {
         }
 
         for (String entry : cloudEvent.getExtensionNames()) {
-            span.setAttribute(entry, cloudEvent.getExtension(entry) == null ? "" : cloudEvent.getExtension(entry).toString());
+            span.setAttribute(entry, cloudEvent.getExtension(entry) != null ? cloudEvent.getExtension(entry).toString() : "");
         }
```

Comment:
It is OK now. The other one has the same problem. 
But in fact, this issue should not be posted as there is no problem in the original code here. So let's wait the maintainer to make the final decision.